misc/netgen/build.sh

```diff
@@ -22,12 +23,23 @@ echo "---------------------"
 
 echo CPPFLAGS="$CPPFLAGS"
 
-./configure \
-	--prefix="${PREFIX}" \
-	--with-tk="${PREFIX}" \
-	--with-tcl="${PREFIX}" \
-	--x-includes="${PREFIX}" \
-	--x-libraries="${PREFIX}"
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/autoconf/build-aux/config.* scripts
```

Comment:
curious why this was necessary? Should we get those updated upstream?

Reply:
Admittedly, I'm trying this on arm64, and the build is old enough that it doesn't even configure on arm64.

I adapted it from https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock which didn't work on the conda version of gnuconfig.

This should probably be fixed upstream as well.